.github/workflows/main.yml

```diff
@@ -31,6 +31,29 @@ jobs:
         run: |
           cargo test --workspace --all-features
       - name: Run tests (without OGG)
+        run: |
+          cargo test --workspace --no-default-features --features="build"
+
+  tests-distribution-libflac:
```

Comment:
It took me a while to understand what "distribution" means in this case.
I was first thinking that something is distributed here, which it's of course not.

Maybe rename it to something with "external"?
Or is there a better word?

Reply:
It was a coin-flip for me between "distribution" and "external"; "external" it is